nifty/pipeline/objectoriented/GetConfig.py

```diff
@@ -72,6 +72,8 @@ def makeConfig(self):
         self.parser.add_argument('-i', '--interactive', dest = 'interactive', default = False, action = 'store_true', help = 'Create a config.cfg file interactively.')
         # Ability to repeat the last data reduction
         self.parser.add_argument('-r', '--repeat', dest = 'repeat', default = False, action = 'store_true', help = 'Repeat the last data reduction, loading saved reduction parameters from runtimeData/config.cfg.')
+        # Specify where downloads come from; either Gemini or CADC.
+        self.parser.add_argument('-c', '--cadc', dest = 'cadc', default = False, action = 'store_true', help = 'Download raw data from Canadian Astronomy Data Centre rather than the Gemini Science Archive.')
```

Comment:
Hmm.  perhaps --data-source with two choices 'CADC' and 'GSA'  (Gemini Science Archive) would make more sense here.  With GSA being the default.  Eventually you could extend NIFTY with other data sources.

Reply:
74a11e9 implements this by changing the "-c" flag to a "-d/--data-source" option with 'GSA' and 'CADC" options. It also adds some better error handling in regards to your next comment.

Reply:
`-d` is typically used for debug mode, at least we've been using it for that.

Reply:
How about -s/--data-source? Implemented in 9214743 but can be changed very easily in one line.